.goreleaser.yml

```diff
@@ -80,7 +83,7 @@ brews:
       - ttn-lw-migrate
     tap:
       owner: TheThingsNetwork
-      name: homebrew-lorawan-stack-migrate
+      name: homebrew-lorawan-stack
```

Comment:
is this correct?

Reply:
Yes, but I'm not sure if this works. The idea is that the file gets _added_ here: https://github.com/thethingsnetwork/homebrew-lorawan-stack. Or would goreleaser replace the contents?

Reply:
Based on the commit history, it looks like GoReleaser adds files, so this should be fine 🤞